primitives/Cargo.toml

```diff
@@ -19,8 +19,10 @@ default = ["std"]
 std = ["alloc", "hashes/std", "internals/std", "io/std", "units/std"]
 alloc = ["hashes/alloc", "internals/alloc", "io/alloc", "units/alloc"]
 serde = ["dep:serde", "hashes/serde", "internals/serde", "units/serde", "alloc"]
+arbitrary = ["dep:arbitrary", "units/arbitrary"]
 
 [dependencies]
+arbitrary = { version = "1", optional = true }
```

Comment:
We can fix this later, but just for your learning, we tend to put optional dependencies down below non-optional ones separated by a line of whitespace.